core/src/main/java/nl/nn/adapterframework/metrics/FrankDistributionSummary.java

```diff
@@ -0,0 +1,163 @@
+/*
+   Copyright 2022 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.metrics;
+
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.DoubleAdder;
+import java.util.concurrent.atomic.LongAdder;
+
+import io.micrometer.core.instrument.AbstractDistributionSummary;
+import io.micrometer.core.instrument.Clock;
+import io.micrometer.core.instrument.distribution.CountAtBucket;
+import io.micrometer.core.instrument.distribution.DistributionStatisticConfig;
+import io.micrometer.core.instrument.distribution.Histogram;
+import io.micrometer.core.instrument.distribution.HistogramSnapshot;
+import io.micrometer.core.instrument.distribution.TimeWindowFixedBoundaryHistogram;
+import io.micrometer.core.instrument.distribution.TimeWindowMax;
+
+public class FrankDistributionSummary extends AbstractDistributionSummary {
```

Comment:
Ik vind het geen goed idee om eerst de StatisticsKeeper te vervangen door MicroMeter, en dan vervolgens die statisticskeeper over te gaan bouwen in MicroMeter

Reply:
Ik heb niets vervangen, het draait parallel naast elkaar, om aan te tonen dat het niet via de IBasics hoeft, maar ook via de DistributionSummary zelf kan.